corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1,4 +1,3 @@
-'use strict';
```

Comment:
I'm actually not sure, I just added it back in for now just in case [462cd63](https://github.com/dimagi/commcare-hq/pull/35340/commits/462cd63cc2ed55d149e7108ed1195e62f91f0c70)